devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/core/examples/resteasy-reactive-example/codestart.yml

```
@@ -0,0 +1,19 @@
+name: resteasy-reactive-example
+ref: resteasy-reactive
+type: code
+tags: example
+language:
+  base:
+    data:
+      title: RESTEasy Reactive
+      description: |
+        <p>A Hello World RESTEasy Reactive resource</p>
+      resource:
+        class-name: GreetingResource
```

We need to change this to avoid conflict with RESTEasy:
```suggestion
        class-name: ReactiveGreetingResource
```